pkg/etc/wavefront/wavefront-proxy/wavefront.conf.default

```diff
@@ -3,83 +3,60 @@
+# by clicking on the gear icon at the top right of the Wavefront site -> click on your email -> click on "API access".
+#token=
 
-# Comma separated list of ports to listen on for Wavefront formatted data
+####################################################### INPUTS #########################################################
+# Comma separated list of ports to listen on for Wavefront formatted data (Default: 2878)
 pushListenerPorts=2878
-## Comma separated list of ports to listen on for OpenTSDB formatted data
-#opentsdbPorts=4242
-## Comma separated list of ports to listen on for HTTP JSON formatted data
-#jsonListenerPorts=3878
-## Comma separated list of ports to listen on for HTTP collectd write_http data
-#writeHttpJsonListenerPorts=4878
-## Comma separated list of ports to listen on for Graphite pickle formatted data (from carbon-relay)
-#picklePorts=5878
-
-## Which ports should listen for collectd/graphite-formatted data?
-## If you uncomment graphitePorts, make sure to uncomment and set 'graphiteFormat' and 'graphiteDelimiters' as well.
+## Maximum line length for received points in plaintext format on Wavefront/OpenTSDB/Graphite ports. Default: 32KB
+#pushListenerMaxReceivedLength=32768
+## Maximum request size (in bytes) for incoming HTTP requests on Wavefront/OpenTSDB/Graphite ports. Default: 16MB
+#pushListenerHttpBufferSize=16777216
+
+## Graphite input settings.
+## If you enable either graphitePorts or picklePorts, make sure to uncomment and set 'graphiteFormat' as well.
+## Comma separated list of ports to listen on for collectd/Graphite formatted data (Default: none)
 #graphitePorts=2003
+## Comma separated list of ports to listen on for Graphite pickle formatted data (from carbon-relay) (Default: none)
+#picklePorts=2004
 ## Which fields (1-based) should we extract and concatenate (with dots) as the hostname?
 #graphiteFormat=2
-## Which characters should be replaced by dots in the hostname, after extraction?
+## Which characters should be replaced by dots in the hostname, after extraction? (Default: _)
 #graphiteDelimiters=_
+## Comma-separated list of fields (metric segments) to remove (1-based) (Default: none)
+#graphiteFieldsToRemove=3,4,5
 
-## Number of threads that flush data to the server. If not defined in wavefront.conf it defaults to the
-## number of processors (min 4). Setting this value too large will result in sending batches that are
-## too small to the server and wasting connections. This setting is per listening port.
-#flushThreads=4
-
-## Max points per flush. Typically 40000.
-#pushFlushMaxPoints=40000
+## DDI/Relay endpoint: in environments where no direct outbound connections to Wavefront servers are possible, you can
+## use another Wavefront Proxy that has outbound access, to act as a middleman relay and forward all the data
+## received on that endpoint (from Direct Data Ingestion clients and/or other proxies) to Wavefront servers.
+## This setting is a comma-separated list of ports. (Default: none)
```

Comment:
...use another Wavefront Proxy that has outbound access, to act as a middleman relay and forward all the data received on that endpoint (from Direct Data Ingestion clients and/or other proxies) to Wavefront servers. This setting is a comma-separated list of ports. (Default: none)

...use another Wavefront proxy that has outbound access to act as a relay and forward all the data received on that endpoint (from direct data Ingestion clients and/or other proxies) to Wavefront servers.
...